autoscheduler/frontend/src/components/LandingPage/About/About.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {
+  Link, Dialog, Typography, DialogContent, DialogTitle,
+} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import { withStyles } from '@material-ui/core/styles';
+
+import * as about from './About.css';
+
+const About: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = (): void => {
+    setOpen(true);
+  };
+  const handleClose = (): void => {
+    setOpen(false);
+  };
+
+  const WhiteTextTypography = withStyles({
+    root: {
+      color: '#FFFFFF',
+    },
+  })(Typography);
+
+  return (
+    <div className={about.container}>
+      <Link
+        component="button"
+        variant="body2"
+        onClick={handleClickOpen}
+      >
+                About Rev Registration
+      </Link>
+      <Dialog
+        onClose={handleClose}
+        open={open}
+      >
+        <DialogTitle disableTypography className={about.title}>
+          <WhiteTextTypography variant="h6">About</WhiteTextTypography>
+          <IconButton onClick={handleClose}>
+            <CloseIcon style={{ color: 'white' }} />
```

Comment:
Since both the Typography and this icon need to be white, I would just make a `.white` CSS class and use that for both. That way, all of the styling stays in CSS